### PR TITLE
spark package added

### PR DESCRIPTION
**Pull request by dmitriybondr**
@ryane Please take a look, maybe this will be usefull

### CODE REVIEW

**Comment by ryane** [comment]
@dmitriybondr this would be extremely useful! the only thing is that I don't see how it will work with mesos authentication turned on right now (which is the default on Mantl clusters). Let me know if I missed it. I do believe the spark framework supports mesos auth so we just need to figure out how to conditionally enable it.

**Comment by mcapuccini** [comment]
@dmitriybondr this is amazing, I will test it!

**Comment by ryane** [comment]
This looks really close - I think it just needs to support Mesos framework authentication

File: repo/packages/S/spark/0/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "postInstallNotes": "The Apache Spark DCOS Service has been successfully installed!\n\n\tDocumentation: https://spark.apache.org/docs/latest/running-on-mesos.html\n\tIssues: https://issues.apache.org/jira/browse/SPARK",
+  "scm": "https://github.com/apache/spark.git",
+  "images": {
+    "icon-medium": "https://downloads.mesosphere.com/spark/assets/icon-service-spark-medium.png",
+    "icon-large": "https://downloads.mesosphere.com/spark/assets/icon-service-spark-large.png",
+    "icon-small": "https://downloads.mesosphere.com/spark/assets/icon-service-spark-small.png"
+  },
+  "maintainer": "support@mesosphere.io",
```
Comment (langston-barrett) [inline repo/packages/S/spark/0/package.json:9]: I'm not sure we want to list mesosphere as the maintainer.

**Comment by ryane** [comment]
closing in favor of #30. Thanks!

**Pull request closed without merging**